.github/workflows/acceptance_tests_secondary.yml

```diff
@@ -12,6 +12,8 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
 jobs:
   acceptance-tests-secondary:
+    # Skip running scheduled jobs on forks
+    if: (github.event_name == 'schedule' && github.repository == 'uyuni-project/uyuni') || (github.event_name != 'schedule')
```

Comment:
Is it necessary to strictly check `github.event_name == 'schedule'` for our `uyuni` project?

Reply:
you are right! Good point!

Reply:
fixed